### PR TITLE
Test  rocket

### PR DESCRIPTION
**Pull request by osamaashraf6**
## What does this PR do?

This PR is raised to implement the following features: 👍 :

- [x] Add the test for the rocket and the header👌
- [x] Use the react testing library for the testing⤵️ 
- [x] Use the jest to test the piece of the code of the component
- [x] Use the snapshot for testing the component by the renderer💯  

## Description of the task to be completed 👎 

- [x] Add the test for the rocket and the header👌
- [x] Use the react testing library for the testing⤵️ 
- [x] Use the jest to test the piece of the code of the component

  
## How should it be tested

Clone the repository
```
git clone https://github.com/osamaashraf6/Space-X-React-Group
```
Navigate to the folder
```
cd Space-X-React-Group
```
```
Checkout the branch
```
git checkout test__rocket
```

#### Thanks for reviewing my PR 😊

### CODE REVIEW

File: src/components/rocket/singlerocket/singlerocket.js [hunk 1]
```diff
@@ -44,11 +45,11 @@ function SingleRocket({
...
-  description: PropTypes.string.isRequired,
-  images: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string])).isRequired,
-};
+// SingleRocket.propTypes = {
+//   id: PropTypes.number.isRequired,
+//   reversed: PropTypes.bool.isRequired,
+//   rocketName: PropTypes.string.isRequired,
+//   description: PropTypes.string.isRequired,
+//   images: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string])).isRequired,
+// };
```
Comment (momotsow) [inline src/components/rocket/singlerocket/singlerocket.js:54]: Hi @osamaashraf6, Kindly remove the commented-out code

**Review by momotsow — CHANGES_REQUESTED** [review]
## Hi :raising_hand: @osamaashraf6 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


Screenshot from 2022-10-20 09-38-51


## Highlights :green_circle: 

- Each value of the action type was scaled using the ducks pattern :+1: 
- Great commit history :+1: 
- PR is descriptive :+1: 
- All Linter checks passed :heavy_check_mark: 


## Required Changes ♻️
- I tried running the tests on my side and it fails with this message: "**Cannot find module 'react-test-renderer' from 'src/__test__/components.test.js**' "


_Check the comments under the review._

## Optional suggestions
N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by osamaashraf6** [comment]
- Thanks @momotsow 🙏 for reviewing my pull request I appreciate your review I will work on this part to enhance my code thanks for advising me 👌 💯 
### Thanks again for reviewing my PR  and have a good day😊

**Review by momotsow — APPROVED** [review]
Hi @osamaashraf6 , 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the Git Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Comment by osamaashraf6** [comment]
- Thanks @momotsow 🙏 for reviewing my pull request I appreciate your review 💯 
### Thanks again for reviewing my PR  and have a good day😊

**Pull request merged**